src/frontend-pwa/src/components/Header/Header.tsx

```diff
@@ -9,26 +14,20 @@ import {
   Heading,
   Banner,
   Image,
-  StyledSettingsButton,
-  StyledIcon,
 } from './header.styles';
 
 export default function Header() {
   return (
     <HeaderWrapper>
       <Banner>
-        <Image href="https://gov.bc.ca">
+        <Image href="/">
```

Comment:
If this was a styled <a> element but you switched to using an <img> shouldn't you change the styled Image to `img` instead of `a`?

Reply:
@BradyMitch I don't understand what you mean by "using an". Is there a word missing there? The "Image" tag is a styled "a" tag wrapping around the img allowing for a link to be associated with the img. I could be being very dumb, but I'm not sure how else I would do it.

Reply:
The Text box filtered him out
![image](https://github.com/bcgov/citz-imb-wayfinder/assets/62873746/2ca6f88c-ff94-46f0-bec1-05b97cb83365)
😅

Reply:
@BradyMitch Ah yes, that would make more sense. Sorry, that took me way too long to understand what you meant. Assuming what I changed is what you meant